src/pages/EQIPPage.tsx

```diff
@@ -6,7 +6,7 @@ import Drawer from "../components/ProgramDrawer";
 import SemiDonutChart from "../components/SemiDonutChart";
 import DataTable from "../components/eqip/EQIPTotalTable";
 import EqipTotalMap from "../components/eqip/EqipTotalMap";
-import chartData from "../data/eqip/EQIP_STATUTE_PERFORMANCE_DATA.json";
+import chartData from "../data/EQIP/EQIP_STATUTE_PERFORMANCE_DATA.json";
```

Comment:
```suggestion
import chartData from "../data/eqip/EQIP_STATUTE_PERFORMANCE_DATA.json";
```
I think this needs to be updated back to small caps to fix the test error.